docs/INSTALL.md

```diff
@@ -0,0 +1,51 @@
+# Install Stargz Snapshotter
+
+We assume that you are using containerd (> v1.4.2) as a CRI runtime.
+
+## Install Stargz Snapshotter with Systemd
+
+- Download release tarfile from [the release page](https://github.com/containerd/stargz-snapshotter/releases). For example, amd64 binary of v0.5.0 is available from https://github.com/containerd/stargz-snapshotter/releases/download/v0.5.0/stargz-snapshotter-v0.5.0-linux-amd64.tar.gz.
+
+- Add the following configuration to containerd's configuration file (typically: /etc/containerd/config.toml). Please see also [an example configuration file](https://github.com/containerd/stargz-snapshotter/blob/master/script/config/etc/containerd/config.toml).
+  ```
+  version = 2
+
+  # Enable stargz snapshotter for CRI
+  [plugins."io.containerd.grpc.v1.cri".containerd]
+    snapshotter = "stargz"
+    disable_snapshot_annotations = false
+
+  # Plug stargz snapshotter into containerd
+  [proxy_plugins]
+    [proxy_plugins.stargz]
+      type = "snapshot"
+      address = "/run/containerd-stargz-grpc/containerd-stargz-grpc.sock"
+
+  ```
+
+- Install fuse
+
+  ###### centos
+  ```
+  # centos 7
```

Comment:
Do we support CentOS 7 (kernel 3.10)?

Reply:
I think we don't need this.

Reply:
Please let me know why it's not supported on  CentOS 7 (kernel 3.10), I will add  description about it

Reply:
@ktock @AkihiroSuda Please take a look. Should I remove the script about Centos 7?

Reply:
containerd doesn't explicitly finishes centos 7 support so let's keep it for now.
@AkihiroSuda SGTY?

Reply:
@ktock Either is fine to me

Reply:
I have tested stargz in CentOS 7 (kernel 3.10). It works. I will add back this script